jvb/src/main/java/org/jitsi/videobridge/EndpointMessageTransport.java

```diff
@@ -126,7 +127,14 @@ public BridgeChannelMessage clientHello(ClientHelloMessage message)
         // remote endpoint and the Videobridge is operational.
         // We take care to send the reply using the same transport channel on
         // which we received the request..
-        return new ServerHelloMessage();
+        if (config.announceVersion())
+        {
+            return new ServerHelloMessage(endpoint.getConference().getVideobridge().getVersion().toString());
```

Comment:
`endpoint.getConference().getVideobridge().getVersion().toString()` is definitely unfortunate...but this still probably represents a net win compared to the old method of version announcement.